### ISSUE TITLE
Release v0.5.0 to NPM as samples/package.json already using it

### ISSUE THREAD

**Issue by artemyarulin** [googleapis/nodejs-monitoring#31]
Last available version in NPM is 0.4.1, and simple git clone of this project, with `cd samples && npm install` is failing as it uses 0.5.0 already. Not a big deal, but would be nice to have it in sync and cloudshell would be easier to use

**Comment by stephenplusplus** [comment]
It looks like our CI-driven release is blocked by failing tests. There's an issue to get those fixed here: https://github.com/googleapis/nodejs-monitoring/issues/31

**Comment by stephenplusplus** [comment]
(cc @alexander-fenster)

**Comment by artemyarulin** [comment]
Yeah, no rash - I can edit package.json always, just wanted to let you know, thank you :)
